### PR TITLE
Add Python virtual env to gitignore

### PR DESCRIPTION
**Pull request by antoniotorresm**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,6 @@ src/ipa-tuura/scimv2bridge/migrations/
 
 # In-tree build files
 *~
+
+# Virtual environment
+venv/
```
Comment (f-trivino) [inline .gitignore:16]: @antoniotorresm what about adding:

# Environments
.env
.venv
env/
venv/
ENV/
env.bak/
venv.bak/

Comment (f-trivino) [reply .gitignore:16]: and we can also add entries for the doc venv:
```
docs/.venv
...
```

Comment (antoniotorresm) [reply .gitignore:16]: Updated

Comment (f-trivino) [reply .gitignore:16]: let's take this opportunity to add:

/doc/_build
/doc/.venv


WDYT?

Comment (antoniotorresm) [reply .gitignore:16]: We have `/docs/_build` and `/docs/.venv`, do we want `/doc/*` as well?

Comment (f-trivino) [reply .gitignore:16]: ohh.. you are right...
my bad.
no, we are fine with what it is already there.

**Review by f-trivino — APPROVED** [review]
LGTM.

**Pull request merged**